parts/k8s/containeraddons/kubernetesmasteraddons-blobfuse-flexvolume-installer.yaml

```diff
@@ -29,11 +29,11 @@ spec:
         imagePullPolicy: Always
         resources:
           requests:
-            cpu: <cpuReq>
```

Comment:
While we're doing this change, can we remove the static `andyzhangx/blobfuse-flexvol-installer` config (above) and wire it up for `ContainerImage` template func usage?

Reply:
Sure!